src/ray/core_worker/transport/direct_actor_task_submitter.h

```diff
@@ -61,6 +63,8 @@ class CoreWorkerDirectActorTaskSubmitterInterface {
 
   virtual void CheckTimeoutTasks() = 0;
 
+  virtual void SetPreempted(const ActorID &actor_id) = 0;
```

Comment:
```suggestion
  /// Mark that the corresponding actor is preempted (e.g., spot preemption).
  /// If you call this API, when an actor is dead, it will include preempted = true to death cause. 
  virtual void SetPreempted(const ActorID &actor_id) = 0;
```